### PR TITLE
Yield repositories. Basic impl

### PR DESCRIPTION
**Pull request by ValeriyaMakusheva**
## Description

(Add description of what was added in this PR)

## Issue
Link to issue

## Testing scenarios

- [ ] Scenarios 1: Make sure migration was applied
- [ ] Scenarios 2: Request /v1/me returns {"some":"object}
- [ ] Scenarios 3: When you create user you should not be able to use the same email 2 times

## Impact

(Add short description of what is effect and what can be effect with this feature/release)

## Rollback

(What and how to rollback the code.  Revert commit and run this sql to undo migration)

### CODE REVIEW

File: src/main/java/com/change/yieldservice/database/entity/accrued_yield/AccruedYield.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+import com.change.yieldservice.database.entity.AbstractEntity;
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.time.Instant;
+import java.util.Objects;
+import java.util.UUID;
+import org.springframework.data.relational.core.mapping.Table;
+
+@Table("accured_yield")
```
Comment (juriem-changeinvest) [inline src/main/java/com/change/yieldservice/database/entity/accrued_yield/AccruedYield.java:11]: Please change name to plural form in database where it is possible.

File: src/main/java/com/change/yieldservice/database/entity/strategy/Strategy.java [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+import java.math.RoundingMode;
+import java.util.Objects;
+import org.springframework.data.relational.core.mapping.Column;
+import org.springframework.data.relational.core.mapping.Table;
+
+@Table("strategies")
+public class Strategy extends AbstractEntity {
+  private String name;
+
+  @Column("yield")
```
Comment (juriem-changeinvest) [inline src/main/java/com/change/yieldservice/database/entity/strategy/Strategy.java:15]: Not critical, but why not to use yield_percent? We can update technical document later. Same for next fields.

File: src/main/java/com/change/yieldservice/database/entity/Builder.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
+package com.change.yieldservice.database.entity;
+
+import java.lang.reflect.InvocationTargetException;
+import java.util.function.Consumer;
+
+public class Builder<T> {
```
Comment (juriem-changeinvest) [inline src/main/java/com/change/yieldservice/database/entity/Builder.java:6]: As I understand this one used for testing purpose. Please move it to test namespace.

Comment (ValeriyaMakusheva) [reply src/main/java/com/change/yieldservice/database/entity/Builder.java:6]: We can use it in the code also

**Review by henrykukk-change — APPROVED** [review]

**Pull request merged**